src/functionalTest/resources/features/opalMode/PO-184-187-188-192_addNote_postedBy_postedByAAD.feature

```diff
@@ -9,7 +9,7 @@ Feature: tests for changes to postedBy and postedByAAD
     And the add notes response contains
       | associatedRecordId | 500000010                                |
       | noteText           | test postedBy and PostedByAAD Opal user1 |
-      | postedBy           | L070KG                                   |
+      | postedBy           | L017KG                                   |
```

Comment:
This has only changed because the roles are current stored in a 'Set', so return order is non-deterministic